internal_transfers/actions/scripts/backfill.ts

```diff
@@ -7,6 +7,8 @@ const dotenv = require("dotenv");
 
 dotenv.config();
 
+const dateRegex = /^\d{4}-\d{2}-\d{2}$/;
```

Comment:
Please use upper case snake case for constants

```suggestion
const DATE_REGEX = /^\d{4}-\d{2}-\d{2}$/;
```

Reply:
Also, I just realized you used a Date Regex that doesn't really capture as much as it could. The issue describes something a bit more involved.

```
^\d{4}\-(0[1-9]|1[012])\-(0[1-9]|[12][0-9]|3[01])$
```

I would still accept the PR with the simpler one, but wanted to mention...